interfaces/kmod/backend.go

```diff
@@ -64,18 +64,14 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 func (b *Backend) Setup(snapInfo *snap.Info, confinement interfaces.ConfinementOptions, repo *interfaces.Repository) error {
 	snapName := snapInfo.Name()
 	// Get the snippets that apply to this snap
-	snippets, err := repo.SecuritySnippetsForSnap(snapInfo.Name(), interfaces.SecurityKMod)
+	spec, err := repo.SnapSpecification(b.Name(), snapName)
 	if err != nil {
 		return fmt.Errorf("cannot obtain kmod security snippets for snap %q: %s", snapName, err)
```

Comment:
From the mount backend:
"return fmt.Errorf("cannot obtain mount security snippets for snap %q: %s", snapName, err)"
;)
How about: "cannot obtain kmod specification for snap %q: %s"... ?